packages/appsync-emulator-serverless/lambdaSource.js

```diff
@@ -0,0 +1,43 @@
+const path = require('path');
+const { fork } = require('child_process');
+const e2p = require('event-to-promise');
+
+const Runner = path.join(__dirname, 'lambdaRunner');
+
+const lambdaSource = async (
+  { serverlessConfig: { functions }, serverlessDirectory },
+  fn,
+  { payload },
+) => {
+  const fnConfig = functions[fn];
+  if (!fnConfig) {
+    throw new Error(`Cannot find function config for function : ${fn}`);
+  }
+
+  const [handlerPath, handlerMethod] = fnConfig.handler.split('.');
+  const fullPath = path.join(serverlessDirectory, handlerPath);
+
+  const child = fork(Runner, [], {
+    stdio: [0, 1, 2, 'ipc'],
+  });
+  child.send({
+    module: fullPath,
+    handlerPath,
+    handlerMethod,
+    payload,
+  });
+
+  const response = await e2p(child, 'message');
```

Comment:
Should this be moved to L23?

Reply:
No this is fine as events are always going to be triggered by the next tick in the event loop.